src/backend/api/reservations.js

```diff
@@ -0,0 +1,77 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+//Returns all reservations
+router.get("/", async (request, response) => {
+  try {
+    const allReservations = await knex("reservation");
```

Comment:
maybe it works but this seems more readable format
knex
  .from('books')
  .select('title', 'author', 'year')

But this is just my personal opinion :)